index.js

```diff
@@ -17,7 +17,7 @@ var Notify = React.createClass({
 	addNotify: function(title, msg, time, theme) {
 		var key = this.key++;
 		this.state[key] = { title: title, msg: msg, time: time, theme: theme };
-		this.setState(this.state);
+		this.forceUpdate();
```

Comment:
Realistically you shouldn't directly be modifying `this.state`. If you do, there isn't really much difference in calling `this.setState` or `this.forceUpdate`. 

Alternatively, consider passing a function through to `this.setState` -
```js
this.setState(function (prevState, props) {
   prevState[key] = {}; // whatever here
   return prevState;
});
```